api/types/constants.go

```diff
@@ -661,6 +661,10 @@ const (
 	ReqAnnotationSchedulesLabel = "/schedules"
 	// ReqAnnotationNotifyServicesLabel is the request annotation key at which notify services are stored for access plugins.
 	ReqAnnotationNotifyServicesLabel = "/notify-services"
+	// ReqAnnotationApprovedCloseCode is the request annotation key at which the servicenow close code for approved access requests is stored.
```

Comment:
Can you please describe a bit what's the usecase for these labels and why they are required?
It's not clear to me what they are used for when reading the comment.